src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(llvm_asm)]
```

Comment:
`#![feature(..)]` is not allowed in the stable channel so this should also be behind the "inline-asm" Cargo feature. You'll need to write something like this:

``` rust
#![cfg_attr(feature = "inline-asm", feature(llvm_asm))]
```

Also you should move this to after the crate level doc comment so it sits next to the other crate level attributes.

Reply:
Ah my bad. I made a mistake here, I thought I had compiled this with the stable toolchain before committing, turns out I did not.